core/src/main/scala/chandu0101/scalajs/react/components/ReactTable.scala

```diff
@@ -126,35 +120,30 @@ object ReactTable {
           S.filteredModels.length,
           S.offset,
           onNextClick,
-          onPreviousClick
-        )
-      )
+          onPreviousClick))
   }
 
-  def getIntSort(key: String) = (m1: Model, m2: Model) => m1(key).asInstanceOf[Int] < m2(key).asInstanceOf[Int]
+  def getIntSort(key: String) = (m1: Model, m2: Model) ⇒ m1(key).asInstanceOf[Int] < m2(key).asInstanceOf[Int]
```

Comment:
Yeah, i know. Anyways, we could change *all* the arrows, but if we do, lets do it in another PR